pkg/features/kube_features.go

```diff
@@ -98,7 +94,6 @@ func init() {
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubernetes binaries.
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
-	ExternalTrafficLocalOnly:                    {Default: true, PreRelease: utilfeature.GA},
```

Comment:
Specifically, any user who had set this key, regardless of the value they chose, will have their components suddenly fail (I am pretty sure).

Maybe we should change the feature-gate lib to just warn on unknown keys, or we should leave keys in place but mark them obsolete (and not show them in help)